test/index.js

```diff
@@ -1,5 +1,59 @@
+/* eslint-disable max-len */
+
+const cli = require('../lib/index')
```

Comment:
nit: how we require in tests should match as closely to how consumers will require.  So this really should be `require('..')` so that the `main` entry in `package.json` is interpreted.  I'm not sure how we'd want to require the server methods.

Reply:
Agreed thanks for pointing this out